Sources/Penny/Services/DiscordService/DiscordService.swift

```diff
@@ -294,22 +292,6 @@ actor DiscordService {
             return []
         }
     }
-    
-    func getPossiblyCachedChannelMessage(
-        channelId: ChannelSnowflake,
-        messageId: MessageSnowflake
-    ) async -> DiscordChannel.Message? {
-        if let cached = self.cachedMessages[[AnySnowflake(channelId), AnySnowflake(messageId)]] {
-            return cached
-        } else {
-            if let message = await getChannelMessage(channelId: channelId, messageId: messageId) {
-                self.cachedMessages[[AnySnowflake(channelId), AnySnowflake(messageId)]] = message
-                return message
-            } else {
-                return nil
-            }
-        }
-    }
```

Comment:
`getChannelMessage` already looks at `DiscordCache` for cached messages, so it's no longer worth it to have this function although it could at times cache a message which isn't available in `DiscordCache` (e.g. because the bot has had a restart and has lost its previous cached messages).